tiff-2-png-converter.py

```diff
@@ -8,189 +9,273 @@
 import tifffile  as tiff
 import numpngw
 from progress_bar import *
+from image_process_helper import *
```

Comment:
Wildcard imports are generally discouraged. Consider importing just the functions you need or the module with as shorter alias to keep traceability

Reply:
removed star and added function used from that file

Reply:
I still the a star on line 11, could you update that one as well?